migaloo/assetlist.json

```diff
@@ -382,6 +382,49 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/lsdshark.png"
         }
       ]
+    },
+    {
+      "description": "Rakoff Token",
+      "denom_units": [
+          {
+              "denom": "ibc/44C29C91F202E20C8E28DFB1FA89B725C54171CD77B8948836C72E7A97E4A018",
+              "exponent": 0
+          },
+          {
+              "denom": "Rakoff",
+              "exponent": 6
+          }
+      ],
+      "base": "ibc/44C29C91F202E20C8E28DFB1FA89B725C54171CD77B8948836C72E7A97E4A018",
+      "address": "cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2",
+      "type_asset": "ics20",
+      "name": "Rakoff",
+      "display": "Rakoff",
+      "symbol": "Rakoff",
+      "traces": [
+        {
+          "type": "ibc-cw20",
+          "counterparty": {
+            "chain_name": "terra",
+            "base_denom": "cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2",
```

Comment:
This asset is not registered. Please define it in the Terra Classic Assetlist

Reply:
Please, refer to https://github.com/cosmos/chain-registry/pull/4292

Reply:
#4287 has been merged - thank you.